docs/src/phyto_equations.md

```diff
@@ -149,42 +149,100 @@ chl &= chl + S_{chl} \cdot \Delta T \nonumber
 
 ## Cell division
 
-Cell size, ``Sz``, is used to indicate cell division. For the smallest cell, ``Sz=1.0`` and ``Bm=Cquota``. Cells start to divide at ``Sz=2.0`` and the probability of individual cell division is then given by a sigmoidal function of ``Sz``.
+Relative cell size, ``Sz``, is used to indicate cell division. Cells will not divide if ``Sz<2.0`` and the probability of individual cell division (``P_D``) is then given by a sigmoidal function of ``Sz`` with different cell division strategies.
 
 ```math
 Sz = (Bm + Cq) / Cquota
 ```
 
 ```math
-P_{divide} = rand(Bernoulli(P_{dvid}*(tanh(stp_{dvid}*(Sz-b))+1)))
+P_D = rand(Bernoulli(P_{dvid}*S_{dvid}))
 ```
 
-``P_{divide}`` is computed every hour no matter what time step the model is.
+where ``S_{dvid}`` represents the sigmoidal function in different cell division strategies (listed below) and ``P_D`` is computed every 10 min (when model time step is smaller than 10 min) or every time step (when model time step is larger than 10 min).
+
+### Sizer
+
+The probability of cell division is a function of cell size and increase while the cell size gets larger.
+
+```math
+S_{dvid} = tanh(stp_D * (Sz - reg_D)) + 1.0
+```
+
+### Adder
+
+The probability of cell division is a function of the difference between current cell size and initial cell size (``iSz``) and increase while the difference gets larger.
+
+```math
+S_{dvid} = tanh(stp_D * (Sz - iSz - reg_D)) + 1.0
+```
+
+### Timer
+
+The probability of cell division is a function of clock time and increase in the dark.
+
+```math
+S_{dvid} = tanh(stp_D * (t - reg_D)) + 1.0
+```
+
+### Sizer+Timer
+
+The probability of cell division is a multiplication of a function of clock time and a function of cell size.
+
+```math
+S_{dvid} = (tanh(stp1_D * (Sz - reg1_D)) + 1.0) * (tanh(stp2_D * (t - reg2_D)) + 1.0)
+```
+
+### Adder+Timer
+
+The probability of cell division is a multiplication of a function of clock time and a function of the difference of current cell size and initial cell size.
+
+```math
+S_{dvid} = (tanh(stp1_D * (Sz - iSz - reg1_D)) + 1.0) * (tanh(stp2_D * (t - reg2_D)) + 1.0)
+```
+!!! tip "Parameters of cell division"
+    ``stp1_D``, ``reg1_D``, ``stp2_D``, and ``reg2_D`` may be different for different cell division strategies.
+
+## Grazing
+
+For now, there're no real grazers in the model. Instead, a quadratic grazing term is implemented in the model to represent the grazing loss of cells. For each individual, the probability to be grazed (``P_G``) is calculated as below.
```

Comment:
maybe replace "there're no real grazers in the model" with "the model does not have a grazer population" if that's what you meant